pkg/service/dbaas/service.go

```diff
@@ -89,6 +99,10 @@ func (s *Service) fetchManagedDBaaS(ctx *Context) error {
 	for _, groupVersionResource := range groupVersionResources {
 		managedResources, err := s.k8sClient.Resource(groupVersionResource).List(ctx, metav1.ListOptions{})
 		if err != nil {
+			// fixme
+			if err.Error() == "the server could not find the requested resource" {
+				continue
+			}
```

Comment:
What exactly should be fixed here?

If it's the matching on error message then I agree. AFAICT you should be able to just use `errors.IsNotFound` from `apimachinery` here or am I missing something?

Reply:
exactly what you just proposed. THanks for the tip :)

Reply:
@glrf I didn't have time to finish it yesterday 😅

Reply:
I realized that afterwards.  Sorry I thought this was done when I reviewed it :see_no_evil:

Reply:
hehe, no problem, I forgot to add the `wip` tag until later too 😄